packages/mock-block-dock/dev/DevApp.tsx

```diff
@@ -5,10 +5,28 @@ import { MockBlockDock } from "../src/MockBlockDock";
 
 const node = document.getElementById("app");
 
-const DevApp = () => (
-  <MockBlockDock>
-    <TestBlock entityId="test-entity-1" name="World" />
-  </MockBlockDock>
-);
+const DevApp = () => {
```

Comment:
This change was to eliminate the possibility that it was the props being passed which were not updating.

It's good to have this in here to display `mock-block-dock` updating to prop changes.

Reply:
I'm not sure about including this - it feels like something that would be better as a test, because it's something of  a secondary use of `mock-block-dock`.

Why do we want to display it updating to prop changes here?

Reply:
I think it's a good to have to display `mock-block-dock`s prop updating abilities as well, since that would be something users would be wondering about.

If not this, we could add a small note on how prop updating happens in `mock-block-dock` maybe? Let me know what you think.

Reply:
A small note sounds good - could you add something to the README that explains something like "If the component's props are changed from outside `MockBlockDock` (e.g. updated from some other state store), the mock entity store will be updated to include the new properties"

Reply:
Have reverted this change and added the note here: https://github.com/blockprotocol/blockprotocol/pull/171/files#diff-9a539852eacb254f155dd2d9ebf9b135cd5130b0e5945a7b7d7d027aaa0628f9R39